readme.md

```diff
@@ -1 +1,77 @@
 Hello!
+
+# li
+
+> A crawler that scrapes COVID-19 Coronavirus data from government and
+  curated data sources.
+
+This project exists to scrape, de-duplicate, and cross-check
+county-level data on the COVID-19 coronavirus pandemic.
+
+Every piece of data includes GeoJSON and population data, cites the
+source from which the data was obtained, and includes a rating of the
+source's technical quality (completeness, machine readability, best
+practices -- not accuracy).
+
+## Where's the data?
+
+TBD
+
+## How often is it updated?
+
+Constantly.
+
+TBD.
+
+## How do I use this data?
+
+TBD ... Read the [Data Fields](./docs/data_fields.md) documentation for
+details on exactly what each field in the dataset means.
+
+## How can I run the crawler locally?
+
+Check out our [Getting Started](./docs/getting_started.md) guide to
+help get our project running on your local machine.
+
+## Contributing
+
+You can contribute to this project in two big ways:
+
+### Contribute to the project core
+
+Check the Issues for any task we need to get done. If you are new to
+open source, look for the label [`Good first
+issue`](https://github.com/covidatlas/li/labels/good%20first%20issue)
+
+### Contribute a source
+
+Contributions for any place in the world are welcome. See the
+[community-curated list of verified data
+sources](https://docs.google.com/spreadsheets/d/1T2cSvWvUvurnOuNFj2AMPGLpuR2yVs3-jdd_urfWU4c/edit#gid=0)
+to find a new datasource to add, and be sure to update the "Scraped?"
+column when you do.
+
+TBD ... To help you contribute a new source, please read the [Sources and
+Scrapers](./docs/sources.md) guide before you start!
+
+Send a pull request with your scraper, and be sure to run the scraper
+first with the instructions specified in the guide to make sure the
+data is valid.
+
+## License
+
+TBD ... This project is licensed under the permissive [BSD 2-clause license](LICENSE).
```

Comment:
Partial to Apache 2.0 fwiw

Reply:
If we want attribution (see below) would should also add CC SA.

Reply:
I'll leave this section as TBD until we decide.

Reply:
Apache.